### PR TITLE
qtcreator: add page

### PR DESCRIPTION
**Pull request by marchersimon**

### CODE REVIEW

File: pages/common/qtcreator.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+- Start Qt Creator but don't load any plugins:
+
+`qtcreator -noload {{all}}`
+
+- Start Qt Creator in presentation mode with pop-ups for key combos:
+
+`qtcreator -presentationMode`
+
+- Start Qt Cretor and show a specific commit hash:
```
Comment (sbrl) [inline pages/common/qtcreator.md:26]: typo?
```suggestion
- Start Qt Creator and show a specific commit hash:
```

File: pages/common/qtcreator.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+- Start Qt Creator but don't load the specified plugin:
+
+`qtcreator -noload {{plugin}}`
+
+- Start Qt Creator but don't load any plugins:
+
+`qtcreator -noload {{all}}`
+
+- Start Qt Creator in presentation mode with pop-ups for key combos:
```
Comment (bl-ue) [inline pages/common/qtcreator.md:22]: Is this still accurate?
```suggestion
- Start Qt Creator in presentation mode with pop-ups for keyboard shortcuts:
```

File: pages/common/qtcreator.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+`qtcreator -presentationMode`
+
+- Start Qt Creator and show a specific commit hash:
+
+`qtcreator -git-show {{commit_hash}}`
```
Comment (bl-ue) [inline pages/common/qtcreator.md:28]: This description is a bit confusing.  "show a specific commit hash" doesn't necessarily mean anything to infrequent users of the command 🤔

**Review by sbrl — APPROVED** [review]
Looks ok to me after @bl-ue's comments are addressed. Thanks, @marchersimon!

File: pages/common/qtcreator.md (hunk 3 above)
Comment (marchersimon) [reply pages/common/qtcreator.md:28]: I just tested it and it seems to show the diff between the specified commit and the previous one. I don't really know how to correctly say that in "git" language tough.

Comment (navarroaxel) [reply pages/common/qtcreator.md:28]: I think we use `diff` to say that ([git-show.md](https://github.com/tldr-pages/tldr/blob/master/pages/common/git-show.md)):
```
- Start Qt Creator and show the diff from a specific commit:

`qtcreator -git-show {{commit}}`
```

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**